### PR TITLE
[#IP-404] Updated spid certs and metadata

### PR DESCRIPTION
**Pull request by michaeldisaro**
#### List of Changes
- updated certs for spid-testenv
- updated metadata for spid-testenv

#### Motivation and Context
We want to update certs for spid-testenv because previous ones are expired.

#### How Has This Been Tested?
It has been tested by performing a yarn start and checking that spid-testenv allows for sign in.

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by gquadrati** [comment]
Could you add a new paragraph in README to explain how to use this new feature?

**Review by gquadrati — COMMENTED** [review]
I'm not able to regenerate them in proper folder, I just left a few comments

File: io-backend/scripts/generate-spid-certs.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+docker build -t italia-backend/tools:1.0.0 docker/images/tools
+docker run --rm -v "$PWD:/usr/src/app" -e "NODE_ENV=development" -w "/usr/src/app" italia-backend/tools:1.0.0 yarn build
```
[the remark this answers is not in the document]
Comment (michaeldisaro) [reply io-backend/scripts/generate-spid-certs.sh:4]: Yeah right... forgot that... I fix a bit the structure of this barbatrick.

[the remark this answers is not in the document]
Comment (michaeldisaro) [reply io-backend/scripts/generate-spid-certs.sh]: Cleared in https://github.com/pagopa/io-mock/pull/46/commits/6c3fe743ab5ffdab6ac0bcd7e62aebdcda509c6a

File: io-backend/scripts/package.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "scripts": {
+    "generate:test-certs": "./_generate-test-certs.sh ."
```
[the remark this answers is not in the document]
Comment (michaeldisaro) [reply io-backend/scripts/package.json]: Done in https://github.com/pagopa/io-mock/pull/46/commits/6c3fe743ab5ffdab6ac0bcd7e62aebdcda509c6a

**Review by gquadrati — APPROVED** [review]
I tried it, it works great! Thanks!

**Pull request merged**